tests/UI/campaigns/sanity/02_ordersBO/02_editOrder.js

```diff
@@ -11,7 +11,7 @@ const loginCommon = require('@commonTests/loginBO');
 
 // Import pages
 const dashboardPage = require('@pages/BO/dashboard');
-const orderPage = require('@pages/BO/orders/view');
+const orderPage = require('@pages/BO/orders/view/productsBlock');
```

Comment:
Applies here and all the other tests. Maybe choosing a more specific name for the page : 
`productsBlock.js`:  `orderPage` => `orderPageProductsBlock`
`customerBlock.js`:  `orderPage` => `orderPageCustomerBlock`
...

Reply:
Done :+1: